Update Hasura Cloud integration with traces (#1615)
DataDog/integrations-extras: hasura_cloud/README.md

secure, production-ready GraphQL API over your data sources.

The Datadog integration is an observability feature of Hasura Cloud that exports

> your Hasura Cloud project's operation logs, metrics and traces to your Datadog dashboard. 

Reviewer: ```suggestion
your Hasura Cloud project's operation logs, metrics, and traces to your Datadog dashboard. 
```

See the [Datadog Log Facets documentation][4] for information regarding creating facets from logs.

> Logs, metrics and traces from your Hasura Cloud project will be automatically sent to Datadog when your project receives traffic.

Reviewer: ```suggestion
Logs, metrics, and traces from your Hasura Cloud project are automatically sent to Datadog when your project receives traffic.
```

Author: Suggested changes made.
